src/crsAdapter/MerlinAdapter.js

```diff
@@ -192,11 +199,11 @@ class MerlinAdapter {
             }
 
             xmlService.MarkField = service.marked ? 'X' : void 0;
-
-            if ((xmlImport.ServiceBlock.ServiceRow || []).length === 0) {
-                delete xmlImport.ServiceBlock;
-            }
         });
+
+        if (((xmlImport.ServiceBlock || {}).ServiceRow || []).length === 0) {
```

Comment:
I found the condition to be a bit confusing. A try…catch block would be longer, but potentially clearer.

```js
try {
    if (xmlImport.ServiceBlock.ServiceRow.length == 0) {
        delete xmlImport.ServiceBlock;
    }
} catch (e) {}
```

Reply:
Yes, I think the same. On the other side using try catch without error handling feels like "uncomplete code".

Would be nice to have something like the elvis operator from angular ("?.") natively in JS.

But I guess the try-catch variant is the more clean way to do it.
I will change it. Hopefully it will not result in nesting code issues.

Reply:
I've seen the convention of adding `{ /* do nothing */}` to the catch block to make it obvious that errors are intentionally not handled.  Makes it even longer though, leading me to believe that the original version is actually better.

Reply:
I know a version where the variable of the catch block is named to something like "ignored". That will avoid a long try-catch-construct.

Reply:
Oooh, I like that.